extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -592,43 +587,38 @@ function axWFactorySaveVariable() {
 						$return = Wikia::successmsg( "Parse OK, variable saved." );
 					}
 				}
-				ob_end_clean(); #--- puts parse error to /dev/null
+				#--- puts parse error to /dev/null
+				ob_end_clean();
 		}
 
-                if( empty( $error ) ) {
-                    $varInfo = WikiFactory::getVarById($cv_id, $city_id);
-                    if($varInfo->cv_is_unique) {
-                        $wikis = WikiFactory::getCityIDsFromVarValue( $cv_id, $cv_value, '=');
-                        $count = count($wikis);
-                        if( (($count == 1) && ($wikis[0] != $city_id)) || ($count > 1) ) {
-                            $return = Wikia::errormsg( "Value of this variable need to be unique." );
-                            $error++;
-                        }
-                    }
-                }
-
-		wfRunHooks('WikiFactoryVarSave::AfterErrorDetection',array($cv_id,$city_id,$cv_name,$cv_value,&$return,&$error));
```

Comment:
The difference between above and below is from coding standards; spaces -> tabs, spaces in function calls, etc